scalableminds/webknossos
# Show dataset extent in right menu tab

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open a tracing and then open the info tab in the right menu
- it should now display the dataset extent

### Issues:
- fixes #3306

------
